versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-iperf-application-on-an-intra-cluster-slice.mdx

```diff
@@ -67,17 +69,21 @@ The following information is required.
 | <slice configuration\> | The name of the slice configuration file. |
 | <project name\> | The project name on which you apply the slice configuration file. |
 
-Switch the context to the KubeSlice Controller using the following command:
-```
-kubectx <cluster name>
-```
-Run the following command to apply the manifest file on the project namespace:
-```
-kubectl apply -f <slice configuration>.yaml -n <project namespace>
-```
+|Perform these steps:
+
+1. Switch the context to the KubeSlice Controller using the following command:
+    ```
```

Comment:
let the command start just below `S` in switch.

Reply:
updated